app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleSettingsFragment.kt

```diff
@@ -104,12 +103,6 @@ class LocaleSettingsFragment : Fragment() {
 
     override fun onDestroyView() {
         super.onDestroyView()
-
```

Comment:
nit: let's not remove this new line

Reply:
Is there any policy around that? There are 33 cases of overriding the `onDestroy` method, and only 10 of those actually have a new line. It tells me two things: there is no consistency and no line version is dominant.

Reply:
No strong policy or feelings toward it. I only pointed it out because it was a minor (maybe even unintentional?) change (vs lint forcing the change) compare to what was set out to be changed in the commit. An example of an extreme case would be if you did a bunch of refactoring within the same commit, where the refactoring could've been in its own issue/commit for an easier review and separation of concerns between commits. Objectively speaking, whether or not we had this new line would not have an impact on the PR. At best, it's just a personal nit.

That said if I had to reflect on my own personal preference, I would still chose to use new lines to separate out and group logical blocks (if/when blocks, super() from other pieces of logic, initialization from method calls, etc) to make things easier to read, BUT it's definitely okay to keep your changes as-is after is said.

Reply:
Fair enough, changed back. It sounds reasonable not to do partisan-refactoring.